### PR TITLE
Feature/form features

### PR DESCRIPTION
**Pull request by MouhaDiouf**

### CODE REVIEW

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    <%= submit_tag("Submit") %>
+<% end %> -->
+
+<%= form_for @user do |f| %>
+    <%= render 'shared/error_messages'%>
+    <%= f.label :username %> <br>
+    <%= f.text_field :username, placeholder: "Enter your username"%> <br>
+    <%= f.label :email %> <br>
+    <%= f.email_field :email, placeholder: "example@email.com", value:'Email' %> <br>
```
Comment (SunnyAsar) [inline app/views/users/edit.html.erb]: lets get rid of the `value` from the edit form my details are not all returned, instead the default value is.

File: app/views/users/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <%= f.label :password %> <br>
+    <%= f.password_field :password, value:"********"%> <br>
+    <%= f.submit 'Create Account'%>
```
Comment (SunnyAsar) [inline app/views/users/edit.html.erb]: update button still reads create account

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      redirect_to user_path
+    else
+      render action: :edit
+    end
+  end
+
+  def show; end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (mcrd25) [inline app/controllers/users_controller.rb]: It’s better to delete commented code. If you need to review a previous version of your code, use git history.
Please remove

**Comment by Redvanisation** [comment]
Thank you for the code review

**Pull request merged**